src/components/views/settings/tabs/user/VoiceUserSettingsTab.js

```diff
@@ -186,10 +186,13 @@ export default class VoiceUserSettingsTab extends React.Component {
             if (audioInputs.length > 0) {
                 const defaultDevice = getDefaultDevice(audioInputs);
                 microphoneDropdown = (
-                    <Field element="select" label={_t("Microphone")}
-                           value={this.state.activeAudioInput || defaultDevice}
-                           onChange={this._setAudioInput}>
-                        {this._renderDeviceOptions(audioInputs, 'audioInput')}
+                    <Field
```

Comment:
I can't see a functional change in these lines.  To minimise merge issues with the base repository, we need to minimise changes.  We you please revert the changes to the Field tag, unless there is a functional change that I have missed.

Reply:
Done